src/EnergyPlus/MixedAir.cc

```diff
@@ -2277,7 +2261,7 @@ void ProcessOAControllerInputs(EnergyPlusData &state,
             if (!AirNodeFound) {
                 ShowSevereError(state,
                                 format("Did not find Air Node (Zone with Humidistat), {} = \"{}\"",
-                                       state.dataMixedAir->OAController(OutAirNum).ControllerType,
+                                       MixedAirControllerTypeNames[static_cast<int>(state.dataMixedAir->OAController(OutAirNum).ControllerType)],
```

Comment:
@Myoldmopar can comment on use of static_cast with this new enum and whether that's OK for use throughout code. Seems OK here.